README.md

```diff
@@ -9,7 +9,7 @@
 Ark gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes. Ark lets you:
 
 * Take backups of your cluster and restore in case of loss.
-* Copy cluster resources across cloud providers. NOTE: Cloud volume migrations are not yet supported.
+* Copy cluster resources across platforms. This includes cloud providers or on-premises installations. NOTE: Cloud volume migrations are not yet supported.
```

Comment:
I'm thinking it would be clearer to have a separate sentence/para within `Overview` that explicitly says something like "Ark can be used on clusters running on a cloud provider or on-premises.  See our supported platform matrix..."

Separately, I think this line-item itself is already overloaded/slightly confusing - are we saying you can copy things from cluster A to cluster B, or provider A to provider B? (I think we want to say both, but they're 2 separate statements).  Could use some polishing and maybe splitting into two sections